Snowflake.Data/Core/Session/EasyLoggingStarter.cs

```diff
@@ -90,13 +108,34 @@ private string GetLogPath(string logPath)
             var logPathOrDefault = logPath;
             if (string.IsNullOrEmpty(logPath))
             {
-                s_logger.Warn("LogPath in client config not found. Using temporary directory as a default value");
-                logPathOrDefault = Path.GetTempPath();
+                s_logger.Warn("LogPath in client config not found. Using home directory as a default value");
+                logPathOrDefault = _environmentOperations.GetFolderPath(Environment.SpecialFolder.UserProfile);
```

Comment:
I'll wrap it in a try/catch

Reply:
Fixed, getting the home directory is now done through `HomeDirectoryProvider.HomeDirectory()`. If the home directory is null, empty, root `/`, or an error then it will return null

In EasyLoggingConfigFinder, getting a null is not treated as an error. And in EasyLoggingStarter, it will be treated as an error and throw an exception